browser_interface/configuration/templates/configuration/config_tree.html

```diff
@@ -170,5 +170,49 @@
             }
 
         </script>
+        <script>
```

Comment:
These belong in the 
`browser_interface/configuration/static/configuration/js/autowisp.config.tree.js`
file not in the HTML template.